### ISSUE TITLE
Does not iterate though devices in a room or group

### ISSUE THREAD

**Issue by randy32768** [chris1705/homebridge-lightify#13]
When several lights are placed in a room and you ask Siri to turn off the lights in {Room} only one light is turned off.  Same issue with changing temperature or RGB color.  If lights added to a group of accessories, and you ask Siri to act on the group there is the same issue.

**Comment by randy32768** [comment]
I tested the same scenarios with the Homebridge-Lightify-API plug in and don't see the same issues.  My hypothesis is the Lightify bridge can't accept multiple commands asynchronously.  I am suggesting the commands need to be serialized as I see the three separate calls in the debug logs of Homebridge.  If this is the case, it may not be easily fixed in the plugin as the calls are managed by Homebridge.  It is a bit of a shame as I would prefer to talk to the Lightify gateway directly rather than relying on the API service.

**Comment by drjamiwe** [comment]
Yeah I got the same issues. Otherwise its working quiet fine! I would love to see this fixed. Another developer said, that they are using a timer to not send too many requests at once.

https://github.com/rainlake/homebridge-platform-lightify

Maybe you can find info in the code, because this is the only plugin that lets me change the color of my PAR16 RGBW.

Would be highly appreciated!
